js_modules/dagit/packages/core/src/app/ContentRoot.tsx

```diff
@@ -40,6 +41,18 @@ export const ContentRoot = React.memo(() => {
             return <Redirect to={url.replace('/instance', '')} />;
           }}
         />
+        {/* todo dish: These /workspace routes are for backward compatibility. Remove them
+        in November or December 2022. */}
+        <Route path="/workspace" exact render={() => <Redirect to="/definitions" />} />
+        <Route path="/locations" exact render={() => <Redirect to="/definitions" />} />
+        <Route
+          path="/workspace/*"
+          exact
+          render={({match}) => {
+            const {url} = match;
+            return <Redirect to={url.replace('/locations', '')} />;
```

Comment:
Should workspace go to definitions instead of locations which then goes to definitions?

Reply:
edit: I see, the locations list page is gone but the individual location pages are still there.

Reply:
I should also mention, the `/definitions` page won't be there for long. Josh has a new design for Code Locations that will make it unnecessary.

I'm a little confused by my own code here, this might be a typo.

Reply:
Yeah, something got lost in the shuffle. Pushing a fix up.